src/utils.hpp

```diff
@@ -6,8 +6,13 @@
 
 namespace Exiv2::Internal {
 
-constexpr bool startsWith(std::string_view s, std::string_view start) {
+template <typename T, typename Y>
```

Comment:
With the change to templates, now some calls to this function replace `T` with `const char *`

Reply:
Hmm? That would be a hard compilation error.

Reply:
Exactly, like the ones we are getting on CI 😄 
https://github.com/Exiv2/exiv2/actions/runs/4059181741/jobs/6986956283